docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,205 @@
+      <i class="fab fa-google" aria-hidden="true"></i>
+      GCP
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>
+      &nbsp;&nbsp; Azure
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes" class="nav-link">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/vmware-tanzu" class="nav-link">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      VMware Tanzu
+    </a>
+  </li>
+
+<li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>OpenShift</a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises" class="nav-link">
+      <i class="fas fa-building"></i>
+      On-premises
+    </a>
+  </li>
+
+</ul>
+
+This document describes how to configure OpenShift for YugabyteDB universes using Yugabyte Platform. If no cloud providers are configured via the Yugabyte Platform console, the main Dashboard page requests to configure at least one provider.
+
+![Configure Cloud Provider](/images/ee/configure-cloud-provider.png)
+
+To create a YugabyteDB universe using the deployed platform, you start by creating the required Role-based access control (RBAC) and adding the provider in the platform.
+
+## Creating RBAC and kubeconfig
+
+kubeconfig is used by Yugabyte Platform to create universes in the OCP cluster.
+
+To create a ServiceAccount in the yb-platform project, execute the following command:
+
+```shell
+oc apply \
+  -n yb-platform \
+  -f https://raw.githubusercontent.com/yugabyte/charts/master/rbac/yugabyte-platform-universe-management-sa.yaml
+# output
+serviceaccount/yugabyte-platform-universe-management created
+```
+
+The next step is to grant access to this ServiceAccount using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. To create the required RBAC objects, execute the following command:
+
+```shell
+curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
+ | sed "s/namespace: <SA_NAMESPACE>/namespace: yb-platform/g" \
+ | oc apply -n yb-platform -f -
+# output
+role.rbac.authorization.k8s.io/yugabyte-helm-operations created
+role.rbac.authorization.k8s.io/yugabyte-management created
+rolebinding.rbac.authorization.k8s.io/yugabyte-helm-operations created
+rolebinding.rbac.authorization.k8s.io/yugabyte-management created
+```
+
+The next step is to create a kubeconfig for this ServiceAccount. You download a helper script for generating a kubeconfig file by executing the following command:
+
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/YugaByte/charts/master/stable/yugabyte/generate_kubeconfig.py

To generate the kubeconfig file, execute the following:

```

Reply:
@bhavin192 I hit "Commit suggestion" button, but I don't see anything changed in my source file, so I'm not sure what happened. 
How should I change the following text?

The next step is to create a kubeconfig for this ServiceAccount. You download a helper script for generating a kubeconfig file by executing the following command:

```shell
python generate_kubeconfig.py \
 --service_account yugabyte-platform-universe-management \
 --namespace yb-platform
# output
Generated the kubeconfig file: /tmp/yugabyte-platform-universe-management.conf
```

Reply:
I figured it out, sorry for the confusion. 
Here's what we have now:

The next step is to create a kubeconfig for this ServiceAccount. You download a helper script for generating a kubeconfig file by executing the following command:

```shell
wget https://raw.githubusercontent.com/YugaByte/charts/master/stable/yugabyte/generate_kubeconfig.py
```

To generate the kubeconfig file, execute the following:

```shell
python generate_kubeconfig.py \
 --service_account yugabyte-platform-universe-management \
 --namespace yb-platform
# output
Generated the kubeconfig file: /tmp/yugabyte-platform-universe-management.conf
```